src/datadoc/frontend/fields/display_dataset.py

```diff
@@ -300,6 +294,27 @@ class DatasetIdentifiers(str, Enum):
         editable=True,
         id_type=DATASET_METADATA_DATE_INPUT,
     ),
+    DatasetIdentifiers.CUSTOM_TYPE: MetadataInputField(
+        identifier=DatasetIdentifiers.CUSTOM_TYPE.value,
+        display_name="Egendefinert type",
+        description="",
+        disabled=True,
+    ),
+    DatasetIdentifiers.USE_RESTRICTION: MetadataInputField(
```

Comment:
This field should be defined with an enum and dropdown ref: https://statistics-norway.atlassian.net/wiki/spaces/MPD/pages/3039133732/DataDoc+-+Krav+til+dokumentasjon+av+datasett+p+Dapla#bruksrestriksjon%2FuseRestriction